src/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,194 @@
+import React, { FC, useState, forwardRef } from 'react';
+import { keyframes, styled } from '@storybook/theming';
+import * as Acc from '@radix-ui/react-accordion';
+import { color, fontFamily, fontSize, spacing } from '../_tokens';
+import { Icon } from '../Icon';
+import type { Icons } from '../Icon/Icon';
+
+export interface AccordionProps {
+  children: React.ReactNode | string;
+  defaultValue?: string | undefined;
+  iconName?: Icons | false;
+  iconSize?: number;
+  id?: string | undefined;
+  inverse?: boolean;
+  value?: string;
+  triggerCopy?: string;
+}
+
+const slideDown = keyframes`
+  from {
+    height: 0;
+  }
+  to {
+    // dynamically set height
+    height: var(--radix-accordion-content-height);
+  }
+}`;
+
+const slideUp = keyframes`
+  from {
+    height: var(--radix-accordion-content-height);
+  }
+  to {
+    height: 0;
+  }
+}`;
+
+// Outermost accordion container
+const AccordionWrapper = styled(Acc.Root)<{
+  inverse?: boolean;
+}>`
+  border: 1px solid;
+  border-bottom: 0;
+  border-color: ${({ inverse }) =>
+    inverse ? color.whiteTr10 : color.slate200};
+  border-radius: 5px;
+  color: ${({ inverse }) => (inverse ? color.white : color.slate800)};
+  font-family: ${fontFamily.sans};
+  font-size: ${fontSize[16]};
+`;
+
+// Wrapper for each accordion item's trigger & content
+const Item = styled(Acc.Item)<{
+  inverse?: boolean;
+}>`
+  border-bottom: 1px solid;
+  border-color: ${({ inverse }) =>
+    inverse ? color.whiteTr10 : color.slate200};
+  flex-flow: column nowrap;
+  overflow: hidden;
+  padding: ${spacing[2]} ${spacing[8]};
+
+  &:last-of-type {
+    border-bottom-left-radius: 4px;
+    border-bottom-right-radius: 4px;
+  }
+
+  &:focus-within {
+    position: relative;
+    z-index: 1;
+    box-shadow: 0 0 0 2px var(--mauve-12);
+  }
+`;
+
+// the button controlling the open/closed state of each item
+const Trigger = styled(Acc.Trigger)<{
```

Comment:
I would suggest using more realistic data in stories. This helps surface issues that are not obvious. For example, layout on small screens. The text gets centred and wraps, and the icon shrinks.  

Wrapping is probably the right choice but the text should remain left aligned and the icon should remain the same size. 

<img width="283" alt="CleanShot 2023-08-21 at 10 38 14@2x" src="https://github.com/chromaui/tetra/assets/42671/8418d239-5f26-482a-8e93-c6fc9b152e78">